processing/src/main/java/org/apache/druid/jackson/DruidDefaultSerializersModule.java

```diff
@@ -182,5 +184,21 @@ public ByteOrder deserialize(JsonParser jp, DeserializationContext ctxt) throws
         }
     );
     addDeserializer(ResponseContext.class, new ResponseContextDeserializer());
+
+    addSerializer(RowsAndColumns.class, new JsonSerializer<RowsAndColumns>()
+    {
+      @Override
+      public void serialize(
+          RowsAndColumns value,
+          JsonGenerator gen,
+          SerializerProvider serializers
+      ) throws IOException
+      {
+        // It would be really cool if jackson offered an output stream that would allow us to push bytes
```

Comment:
It would seem like it, but switching things around to passing down an InputStream just makes the `WireTransferable` code a lot more complex as now it needs to lazily convert to bytes based on calls to `InputStream`.  It's likely so unwieldy that implementations would just resort to building a `byte[]` and using a BAIS anyway.  The natural way to code it would be to say "write yourself to this stream".